src/main/java/net/snowflake/client/core/SessionUtil.java

```diff
@@ -592,6 +601,10 @@ private static SFLoginOutput newSession(
       HttpUtil.applyAdditionalHeadersForSnowsight(
           postRequest, loginInput.getAdditionalHttpHeadersForSnowsight());
 
+      // Add headers for driver name and version
+      postRequest.addHeader(SF_HEADER_CLIENT_APP_ID, SF_DRIVER_NAME);
```

Comment:
In the request body it was using loginInput.getAppId() and loginInput.getAppVersion(). Can we use the same to be consistant?